resources/dicts/events/death/general.json

```diff
@@ -3612,25 +3752,27 @@
             "classic"
         ],
         "weight": 20,
-        "event_text": "m_c was too weak to recover from an illness, and passed away.",
+        "event_text": "m_c was too weak to recover from an illness and passed away.",
         "m_c": {
             "age": [
-                "senior",
                 "adolescent",
                 "young adult",
                 "adult",
                 "senior adult",
-                "elder"
+                "senior"
             ],
-            "status": [],
+            "status": [ "any" ],
             "dies": true
         },
         "history": {
-            "reg_death": "m_c was killed by sickness."
+            "cats": "m_c",
+            "reg_death": "m_c succumbed to illness.",
+            "lead_death": "succumbed to illness"
         }
     },
     {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_whitecough_leafbare2",
```

Comment:
there were a bunch of kitten specific sickness deaths earlier in here, right? can we just smush them into these?

Reply:
not 100% sure what you mean -- delete the other deaths, or group them together?